class.jetpack-options.php

```diff
@@ -389,7 +398,7 @@ static function update_raw_option( $name, $value, $autoload = false ) {
 				)
 			);
 		}
-		return $updated_num;
+		return (bool)$updated_num;
```

Comment:
Can you please add whitespace between the cast and the var?

Reply:
https://github.com/Automattic/jetpack/pull/11992/commits/2fcb5ba8343eae05a94656ec781c4939a2dff386